src/components/DetailsChart/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+import { Card } from "components/Card";
+import { Chart } from "components/Chart";
+import { ChartData, LiquidityPoolHistory } from "types/types.d";
+
+import "./styles.scss";
+
+interface DetailsChartProps {
+  poolHistory: { data: LiquidityPoolHistory[] };
+}
+
+export const DetailsChart = ({ poolHistory }: DetailsChartProps) => {
+  const [chartData, setChartData] = useState([] as ChartData[]);
+  useEffect(() => {
+    const formattedData = poolHistory.data.map((entry) => ({
+      x: new Date(entry.ts * 1000).getDate().toString(),
+      y: Number(entry.total_value_locked),
+    }));
+    setChartData(formattedData);
+  }, [poolHistory]);
+
+  return (
+    <div className="DetailsChart">
+      <Card>
+        <Chart
+          chartData={chartData}
+          colorRgb={[62, 27, 219]}
```

Comment:
We just removed RGB color variables from SDS. 😅 We could make a helper function to translate hex to RGB. That way we could always use SDS theme colors.